src/docs/product/error-monitoring/issue-owners/index.mdx

```diff
@@ -23,14 +23,15 @@ You define ownership rules per project. To configure ownership rules, navigate t
 Types of matchers available:
 
 1. Path: matches against all file paths in the event's stack trace
-2. URL: matches against the event's `url` tag
-3. Tag: matches against event tags
+2. Module: matches against the module in the event's stack trace
+3. URL: matches against the event's `url` tag
+4. Tag: matches against event tags
 
 The general format of a rule is: `type:pattern owners`
 
 `type`
 
-: Can be either `path`, which matches on paths, `url`, which matches the specified URL, or `tags.TAG_NAME` which matches the specified tag.
+: Can be either `path`, which matches on paths, `module`, which matches on module, `url`, which matches the specified URL, or `tags.TAG_NAME` which matches the specified tag.
```

Comment:
These explanations weren't terribly helpful to start. I'd suggest we either enhance them (better explain what path, module, and URL mean in this context) or remove them altogether.

Reply:
Lol I was thinking similarly. The description really doesn't add much above the keyword.

Reply:
@iProgramStuff - shall we simply remove these descriptions? They seem pretty self explanatory except for module.

Reply:
Yes, I think we should. What do you think of the latest copy? :)